gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -508,6 +530,38 @@ def test_pipeline_read_file_pattern_large(self):
     assert_that(pcoll, asserts.count_equals_to(9900))
     pipeline.run()
 
+  @unittest.skipIf(VCF_FILE_DIR_MISSING, 'VCF test file directory is missing')
+  def test_pipeline_read_all_file_pattern_large(self):
+    pipeline = TestPipeline()
+    pcoll = (pipeline
+             | 'Create' >> beam.Create(
+                 [os.path.join(testdata_util.get_full_dir(), 'valid-*.vcf')])
+             | 'Read' >> ReadAllFromVcf())
+    assert_that(pcoll, asserts.count_equals_to(9900))
+    pipeline.run()
+
+  @unittest.skipIf(VCF_FILE_DIR_MISSING, 'VCF test file directory is missing')
+  def test_pipeline_read_all_multiple_files_large(self):
+    pipeline = TestPipeline()
+    pcoll = (pipeline
+             | 'Create' >> beam.Create(
+                 [testdata_util.get_full_file_path('valid-4.0.vcf'),
+                  testdata_util.get_full_file_path('valid-4.1-large.vcf'),
+                  testdata_util.get_full_file_path('valid-4.2.vcf')])
+             | 'Read' >> ReadAllFromVcf())
+    assert_that(pcoll, asserts.count_equals_to(9900))
+    pipeline.run()
```

Comment:
Make sure this same functionality is tested even when the vcf directory is missing.

Reply:
Done.